addons/hr_recruitment/wizard/applicant_send_mail.py

```diff
@@ -45,5 +56,24 @@ def action_send(self):
                 email_layout_xmlid='mail.mail_notification_light',
                 message_type='comment',
                 partner_ids=applicant.partner_id.ids,
-                subject=self.subject,
+                subject=subjects[applicant.id],
+                attachment_ids=self.attachment_ids.ids
             )
+
+    def action_save_as_template(self):
+        for record in self:
+            model = self.env['ir.model']._get('hr.applicant')
+            model_name = model.name
+            template_name = "%s: %s" % (model_name, tools.ustr(record.subject))
```

Comment:
When there's multiple variables coming into a name like that, though here it might not be needed, you should make it translatable and give variable names (I don't know if every languages use ":")
```suggestion
            template_name = _("%(name)s: %(subject)s", name=model_name, subject=tools.ustr(record.subject))
```